File: result/result_2/2.py
from typing import List
import io
def main():
    user_input = None
    print("Input numbers separated by commas: ")
    user_input = input()
    numbers = []
    ss = user_input.split(',')
    temp = None
    for temp in ss:
        numbers.append(int(temp))
    numbers.sort()
    print("Sorted numbers: ")
    i = 0
    while i < len(numbers):
        print(numbers[i])
        if i < len(numbers) - 1:
            print(",")
        i += 1
    print("\n")
    return 0


if __name__ == '__main__':
    main()

File: test/2.cpp
#include <iostream>
#include <vector>
#include <algorithm>
#include <sstream>
#include <string>
using namespace std;
int main() {
    string user_input;
    cout << "Input numbers separated by commas: ";
    getline(cin, user_input);

    vector<int> numbers;
    stringstream ss(user_input);
    string temp;
    
    // 将输入字符串按逗号分割并转换为整数存入vector
    while (getline(ss, temp, ',')) {
        numbers.push_back(stoi(temp));
    }

    // 排序
    sort(numbers.begin(), numbers.end());

    // 输出排序后的结果
    cout << "Sorted numbers: ";
    for (size_t i = 0; i < numbers.size(); ++i) {
        cout << numbers[i];
        if (i < numbers.size() - 1) {
            cout << ",";
        }
    }
    cout << endl;

    return 0;
}
